projects/ngx-maplibre-gl/src/lib/map/map.service.ts

```diff
@@ -44,6 +45,7 @@ import type {
   MapImageOptions,
 } from './map.types';
 import { keepAvailableObjectValues } from '../shared/utils/functions/object.fn';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
```

Comment:
Is this the right path for import?

Reply:
Yes its the correct package. But I just noticed the subscription will be destroyed because it is added to the subscription list.